### ISSUE TITLE
Phase 2 - Need a way to set mobile or desktop testing from the command line

### ISSUE THREAD

**Issue by 20twenty** [FadiRazouk/Project-PWA#17]
Currently, cypress.config.ts must be edited before running tests in order to select mobile or desktop as shown below. We need a way to run each config from the command line without editing this file.

# (temporary) to test mobile you need to change some config in Cypress.config.ts

change from:
```
  viewportWidth: 1387,
  viewportHeight: 764,
  ```
to:
```
  viewportWidth: 390,
  viewportHeight: 844,
```
and isMobile from :

```
false to true
```

**Comment by FadiRazouk** [comment]
Kinda good news, i found out a way to run in command line, down side it does not look pretty, but hey it does the job 😂
ill update the redme

**Comment by 20twenty** [comment]
Excellent!

Let's just add these to the npm run scripts in package.json like so...

"scripts": {
    "cy:run": "cypress run --browser chrome --headless",
    "cy:parallel": "cypress-parallel -s cy:run -t 4 -d '/cypress/e2e/*.cy.ts'",
    "cypress:gui": "cypress run --browser chrome --no-exit",
    "mobile": "docker run -it -v $PWD:/e2e -w /e2e -e CYPRESS_isMobile=true -e CYPRESS_VIEWPORT_WIDTH=390 -e CYPRESS_VIEWPORT_HEIGHT=844 cypress/included:10.10.0",
    "desktop": "docker run -it -v $PWD:/e2e -w /e2e -e CYPRESS_isMobile=false -e CYPRESS_VIEWPORT_WIDTH=1387 -e CYPRESS_VIEWPORT_HEIGHT=764 cypress/included:10.10.0"
  },
